tensorrt_llm/_torch/auto_deploy/models/custom/modeling_gemma4.py

```
@@ -1378,7 +1378,7 @@ def forward(
 
 class Gemma4ForCausalLM(Gemma4TextPreTrainedModel, GenerationMixin):
     config_class = Gemma4TextConfig
-    _tied_weights_keys = ["lm_head.weight"]
+    _tied_weights_keys = {"lm_head.weight": "model.embed_tokens.weight"}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)